playbooks/cnf/deploy-run-eco-gotests.yaml

```diff
@@ -0,0 +1,121 @@
+# eco_cnf_core_net_frr_image        # string: Link to frr image
+# eco_cnf_core_net_mlb_addr_list    # string: List of mlb addresses
+# eco_cnf_core_net_switch_ip:       # string: switch ip address
+# eco_cnf_core_net_switch_user      # string: Username for accessing the intermediate switch.
+# eco_cnf_core_net_switch_pass      # string: Password for accessing the intermediate switch.
+# eco_worker_label                  # string: Label for the CNF worker role.
+# additional_test_env_variables     # string: Any additional env variables accepted by eco-gotests container
+- name: Set up eco-gotests
+  hosts: bastion
+  gather_facts: true
+  vars:
+    eco_gotests_image: quay.io/ocp-edge-qe/eco-gotests
+    eco_gotest_dir: /tmp/eco_gotests
+    eco_gotests_report_path: "{{ eco_gotest_dir}}/report"
+    test_timeout: "12h"
+    eco_cnf_core_net_frr_image: "{{ frr_image_link | default(undefined) }}"
+    eco_cnf_core_net_mlb_addr_list: "{{ tests_mlb_addr_list | default(undefined) }}"
+    eco_cnf_core_net_switch_ip: "{{ hostvars['switch'].ansible_host | default(undefined) }}"
+    eco_cnf_core_net_switch_user: "{{ hostvars['switch'].ansible_user | default(undefined) }}"
+    eco_cnf_core_net_switch_pass: "{{ hostvars['switch'].ansible_password | default(undefined) }}"
+    eco_worker_label: workercnf
+  tasks:
+    - name: Ensure required variables are defined
+      ansible.builtin.assert:
+        that:
+          - kubeconfig is defined
+          - features is defined
+          - labels is defined
+
+    - name: Ensure test diectory does not exist
+      ansible.builtin.file:
+        path: "{{ eco_gotest_dir }}"
+        state: absent
+
+    - name: Ensure test diectory is present
+      ansible.builtin.file:
+        mode: "0760"
+        path: "{{ eco_gotests_report_path }}"
+        state: directory
+        recurse: true
+
+    - name: Gather cluster network information
+      register: cluster_info
+      environment:
+        K8S_AUTH_KUBECONFIG: "{{ kubeconfig }}"
+      kubernetes.core.k8s_info:
+        api: operator.openshift.io/v1
+        kind: Network
+        name: cluster
+
+    - name: Extract OpenShift version details
+      ansible.builtin.set_fact:
+        version: "{{ cluster_info.resources[0].status.version.split('.')[0] }}.{{ cluster_info.resources[0].status.version.split('.')[1] }}"
+
+    - name: Print cluster version
+      ansible.builtin.debug:
+        msg: "Cluster version: {{ version }}"
+
+    - name: Remove an image
+      containers.podman.podman_image:
+        name: "{{ eco_gotests_image }}"
+        state: absent
+        tag: "v{{ version }}.0"
+
+    - name: Pull a specific version of an image
+      containers.podman.podman_image:
+        name: "{{ eco_gotests_image }}"
+        tag: "v{{ version }}.0"
+
+    - name: Prepare execute shell script
+      ansible.builtin.template:
+        src: run_eco_gotests.j2
+        dest: "{{ eco_gotest_dir }}/eco-gotests-run.sh"
```

Comment:
Where is this script stored?

Reply:
eco_gotest_dir=/tmp/eco_gotests
so it's going to be: **/tmp/eco_gotests/eco-gotests-run.sh**